pkg/sechttp/httpclient.go

```diff
@@ -29,21 +29,19 @@ import (
 // Returns: HTTPResponse, HTTPSecError
 func DispatchHTTPRequest(httpClient utils.HTTPClient, originalRequest *http.Request, username string, connectionID string) (*http.Response, *HTTPSecError) {
 
-	logr.SetLevel(logr.TraceLevel)
-
-	logr.Printf("Request URL: %v %v", originalRequest.Method, originalRequest.URL)
+	logr.Tracef("Request URL: %v %v\n", originalRequest.Method, originalRequest.URL)
```

Comment:
Good to leave these in since we can now change the level of logging. Useful information for developers.